client/components/status/status.js

```
@@ -90,160 +90,184 @@ export default class Status extends React.Component {
+    this.setupCaptivePortal();
+
+    await this.handleAuthFlow();
+  };
+
+  setRememberMe = () => {
     this.setState({
       rememberMe: localStorage.getItem("rememberMe") === "true",
     });
+  };
+
+  preloadDependencies = () => {
```

ESLint

[error] Expected 'this' to be used by class method 'preloadDependencies'.

(class-methods-use-this)